config/optional/block.block.servicesmenu.yml

```diff
@@ -0,0 +1,24 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - system.menu.services-menu
+  module:
+    - system
+  theme:
+    - localgov_theme
+id: servicesmenu
```

Comment:
I would normally rename this file to something like `block.block.localgov_services_menu.yml` so it's clear this is a block provided by LocalGov. The `id` would also need updating the match the machine name for the block (i.e. `localgov_services_menu`).

Otherwise looks fine.